modules/removeAll.js

```diff
@@ -0,0 +1,20 @@
+import '../src/style.css';
+import DisplayTodos from './displayTodos.js';
+
+const clearAllTasks = () => {
+  const clearTasks = document.querySelector('#clear-tasks');
+  clearTasks.innerHTML = '';
+
+  const removeAll = document.createElement('button');
+  removeAll.classList.add('remove-all');
+  removeAll.innerHTML = 'Remove All';
+
+  clearTasks.appendChild(removeAll);
+
+  removeAll.addEventListener('click', () => {
+    localStorage.clear();
+    DisplayTodos();
+  });
+};
```

Comment:
- [ ] the `clear all completed ` should remove only the ones that is `completed: true;` your function removes all the todos please fix that and make sure only the completed one are removed 
![image](https://user-images.githubusercontent.com/61790587/171694271-2875fc61-61ef-4d58-8735-956481063cf9.png)